doc/sphinxman/document_capabilities.py

```diff
@@ -0,0 +1,647 @@
+#!/usr/bin/env python
+
+import ast
+import argparse
+import unicodedata
+from pathlib import Path
+
+from psi4.driver.procrouting.proc_data import method_governing_type_keywords
+from psi4.driver.p4util.exceptions import sanitize_method
+
+# TODO how to handle this?
```

Comment:
Could you elaborate? It isn't clear to me why these specific DFT functionals need this. Are these supposed to be representative of various classes DFT functionals?